src/senselab/audio/tasks/text_to_speech/api.py

```diff
@@ -0,0 +1,38 @@
+"""This module implements some utilities for the text-to-speech task."""
+from typing import List, Optional
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.text_to_speech.huggingface import HuggingFaceTTS
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.language import Language
+from senselab.utils.data_structures.model import HFModel, SenselabModel
+
+
+def synthesize_texts(
+    texts: List[str], 
+    model: SenselabModel = HFModel(path_or_uri="suno/bark", revision="main"), 
```

Comment:
not too important but why this model? I know StyleTTS2.0 has been mentioned before though I don't know how their performances compare

Reply:
Consider it as a placeholder for now. We will need to add more models and experiment with them. I like MARS-TTS the most, but I am waiting for them to confirm that I can integrate their technology under an MIT/Apache license. StyleTTS is of course another good option